opcua.cpp

```diff
@@ -178,6 +178,23 @@ DatapointValue* dpv = NULL;
 		m_background = NULL;
 	}
 	Logger::getLogger()->debug("Data change call for Node %s", nodeId);
+
```

Comment:
This certainly matches the acceptance criterion for the work item but it could cause an application to miss a critical data change such as a change in status or a large change in value.